lib/api/v3/users/users_api.rb

```diff
@@ -47,18 +47,20 @@ def user_transition(allowed)
               fail ::API::Errors::InvalidUserStatusTransition
             end
           end
-        end
 
-        resources :users do
-          helpers ::API::V3::Users::CreateUser
+          def authorize_user_cru_allowed
+            allowed_admin = current_user.admin? && (current_user.active? || current_user.is_a?(SystemUser))
```

Comment:
Why is the `SystemUser` allowed here explicitly?

Reply:
Because I took over the other non-admin global checks and that implied a system user. There was a bug I believe that allowed locked admins to do things. This line is now moving through the `UserAllowedService` as well